src/test/java/com/hubspot/jinjava/lib/fn/UnixTimestampFunctionTest.java

```diff
@@ -28,20 +28,20 @@ public void tearDown() {
 
   @Test
   public void itGetsUnixTimestamps() {
-    JinjavaInterpreter.pushCurrent(
-      new JinjavaInterpreter(
-        new Jinjava(),
-        new Context(),
-        JinjavaConfig.newBuilder().build()
-      )
+    JinjavaInterpreter jinjavaInterpreter = new JinjavaInterpreter(
+      new Jinjava(),
+      new Context(),
+      JinjavaConfig.newBuilder().build()
     );
+    JinjavaInterpreter.pushCurrent(jinjavaInterpreter);
     assertThat(Functions.unixtimestamp())
       .isGreaterThan(0)
       .isLessThanOrEqualTo(System.currentTimeMillis());
     assertThat(Functions.unixtimestamp(epochMilliseconds)).isEqualTo(epochMilliseconds);
     assertThat(Functions.unixtimestamp(d)).isEqualTo(epochMilliseconds);
     assertThat(Functions.unixtimestamp((Object) null))
       .isCloseTo(System.currentTimeMillis(), Offset.offset(1000L));
+    assertThat(jinjavaInterpreter.getErrors()).isEmpty();
```

Comment:
this would have caught the errors being thrown that should have only applied to the filter.